src/modelfree/configs/multi/score.py

```diff
@@ -165,6 +175,59 @@ def mask_observations_with_zeros(exp_name, score):
         score['mask_agent_kwargs'] = {'masking_type': 'zeros'}
         exp_name = 'zero_' + exp_name
 
+    @multi_score_ex.named_config
+    def mask_observations_with_additive_noise(exp_name, score, spec):
+        score = score.copy()
+        score['index_keys'] = ['hashable_mask_agent_kwargs', 'mask_agent_noise']
```

Comment:
A lot of code duplicated between this and the subsequent function; factor out into a helper method? (Basically all the part that sets `score`.)

Reply:
I'm pretty hesitant to add another level of factored-out complexity given how complex these dynamic ray/sacred configurations already are to reason about.